### ISSUE TITLE
Template Profile Layout Check

### ISSUE THREAD

**Issue by Maartenf** [HTML5-Games/html5-games.github.io#4]
![profile-design](https://f.cloud.github.com/assets/4083310/595365/65f1981c-caf9-11e2-9679-fcf4c4bc2960.png)
Please leave feedback.

**Comment by Maartenf** [comment]
This is uncoded yet. It's only a PSD-file.

**Comment by ghost** [comment]
Looks great, though it should be implemented on the [main site](https://github.com/HTML5-Games/gamedev) as it requires database access for storing user information.

**Comment by Maartenf** [comment]
Yes, but it isn't coded yet. First, I need to improve it. Later, I can code it 100% HTML5 valid and responsive.

**Comment by Maartenf** [comment]
Sorry for doubleposting, but the image didn't work...
![profile-design](https://f.cloud.github.com/assets/4083310/595419/36e20426-cafe-11e2-87e8-95cae8c67729.png)

**Comment by logan-r** [comment]
@Maartenf Looks awesome!
@williamvanr Could we combined this project and the gamedev project? We could use [Parse](http://parse.com) instead of a server/database and GitHub pages to host it. If you don't want to, I'm okay with that, though. :D

**Comment by ghost** [comment]
@Logi0 I thought they were the same project. I don't mind using Parse but, in the long run, I think it would be better to use a host other than GitHub pages.

**Comment by logan-r** [comment]
@williamvanr Great! There's nothing stopping us from cloning this repo to a server sometime in the future.

**Comment by ghost** [comment]
@Logi0 Ok, so is this the main project now? Is the blog going to be implemented using Parse too? We can't use Wordpress with GitHub Pages.

**Comment by logan-r** [comment]
Yes and yes.

**Comment by ghost** [comment]
@Logi0 Okay, the only problem I see with all of this is that we can't really make dynamic pages with just HTML. GitHub Pages doesn't allow server-side languages (and the security that comes with them) or URL rewriting, which might be especially useful for hosting games, the blog, and the social side of the site. I don't if Parse can handle all of that, but maybe it can. I am learning about Parse now and don't know it's full capabilities.

**Comment by ghost** [comment]
@Logi0 Should probably read [this](https://www.parse.com/questions/javascript-sdk-security) too on Parse security.
